SPECS/installkernel/installkernel

```diff
@@ -0,0 +1,48 @@
+#!/bin/bash
+# Copyright (c) Microsoft Corporation.
```

Comment:
Not MIT?

Reply:
Other distros that package their own version of this `installkernel` script typically license the file as GPLv2. I marked the package as GPLv2.

Reply:
Will add GPLv2